### ISSUE TITLE
If yarn.lock gets fully regenerated, flow will break with a bad configuration

### ISSUE THREAD

**Issue by sjd78** [oVirt/ovirt-web-ui#602]
Steps to reproduce
1.  rebase your repo
1. yarn install
1. yarn build  # this will work fine
1. rm -r yarn.lock node_modules && yarn install
1. yarn build # this will break
1. git checkout -- yarn.lock && rm -r node_modules && yarn install
1. yarn build # this will work again

Error Message: 
```
  Error in Flow: Invalid Flowconfig

  .flowconfig:11 Unsupported option specified! (unsafe.enable_getters_and_setters)
```

Root cause is https://github.com/facebook/flow/issues/5594.  flow hard fails on the **unsafe.enable_getters_and_setters** config line in `.flowconfig`.

Could be rolled into #558 but could become a problem if `yarn.lock` is updated before then.

**Comment by mareklibra** [comment]
Is fixed within #601

Dne út 15. 5. 2018 16:02 uživatel Scott Dickerson <notifications@github.com>
napsal:

**Comment by mareklibra** [comment]
Seems to be fixed now, @sjd78 please reopen otherwise
